src/test/java/org/sagebionetworks/web/unitclient/widget/doi/CreateOrUpdateDoiModalTest.java

```diff
@@ -104,9 +101,20 @@
 	@Captor
 	ArgumentCaptor<DoiRequest> doiRequestCaptor;
 
+	Project mockProjectEntity;
+	FileEntity mockFileEntity;
```

Comment:
Prefer real objects over mocks since `getClass` is called on these, which has unexpected behavior with mocks.

Reply:
Should probably rename.  Maybe testProjectEntity and testFileEntity (since they are not mocked)?